### PR TITLE
release v0.9.1

### PR DESCRIPTION
**Pull request by iaguis**
:sunny:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+## v0.9.1
+
+v0.9.1 is mainly a bugfix and cleanup release.
+
+- Remove redundant dependency fetching, we're vendoring them now ([#134](https://github.com/appc/docker2aci/pull/134)).
+- Export ParseDockerURL which is used by rkt ([#135](https://github.com/appc/docker2aci/pull/135)).
+- Export annotations so people can use them ouside docker2aci ([#135](https://github.com/appc/docker2aci/pull/135)).
```
Comment (krnowak) [inline CHANGELOG.md]: ou**t**side

**Comment by krnowak** [comment]
One typo, otherwise LFAD if it works with rkt too.

File: CHANGELOG.md (hunk 1 above)
Comment (iaguis) [inline CHANGELOG.md]: :cry:

**Comment by iaguis** [comment]
Compiled and tested with rkt. Works \o/

**Comment by krnowak** [comment]
Sheepit!

**Pull request merged**